packages/token/src/erc1155/erc1155.cairo

```diff
@@ -691,7 +690,7 @@ pub mod ERC1155Component {
                     get_caller_address(), from, token_id, value, data
                 ) == interface::IERC1155_RECEIVER_ID
         } else {
-            src5_dispatcher.supports_interface(account::interface::ISRC6_ID)
+            src5_dispatcher.supports_interface(openzeppelin_account::interface::ISRC6_ID)
```

Comment:
For this, would it make sense just to import `ISRC6_ID`? This seems unnecessarily long. Or even something like `use openzeppelin_account::interface as account_interface;`